static/defaults/Reveal.js.yaml

```diff
@@ -4,11 +4,11 @@
 # More info: https://pandoc.org/MANUAL.html
 reader: markdown
 writer: revealjs
-# NOTE: The next two variables must be false. The reason is that Zettlr must
-# perform additional steps after Pandoc has exported your Markdown document
-# in order for themes to work.
-self-contained: false
-standalone: false
+self-contained: true # Must be true (unless you use a custom template)
+# If standalone is true, this will prompt Pandoc to download all required assets
```

Comment:
I think this should be the description for self-contained. Standalone basically just says "use a template".

Not sure if relevant, but self-contained implies standalone. The next pandoc version will split `self-contained` into `embed-resources` (or something similar, forgot the name) and `standalone`, so control over this will be a bit more fine-grained. But that's probably irrelevant for most users here.

Reply:
Ah okay. Yeah, that is definitely good to know. I mean, in the end this here should just be for people who just want the old behavior back ...